server/blob/blob-api/src/test/java/org/apache/james/blob/api/BucketDumbBlobStoreContract.java

```diff
@@ -53,7 +54,7 @@ default void deleteBucketShouldThrowWhenNullBucketName() {
     default void deleteBucketShouldDeleteExistingBucketWithItsData() {
         DumbBlobStore store = testee();
 
-        Mono.from(store.save(TEST_BUCKET_NAME, TEST_BLOB_ID, SHORT_BYTEARRAY)).block();
+        Mono.from(store.save(TEST_BUCKET_NAME, TEST_BLOB_ID, ELEVEN_KILOBYTES)).block();
```

Comment:
Are the changes in this class still required?

Reply:
Not actually, wrong change from me